metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/ExecutionTimeAggregator.java

```diff
@@ -0,0 +1,107 @@
+package io.dockstore.metricsaggregator.helper;
+
+import static io.dockstore.common.metrics.FormatCheckHelper.checkExecutionDateISO8601Format;
+import static io.dockstore.common.metrics.FormatCheckHelper.checkExecutionTimeISO8601Format;
+
+import io.dockstore.metricsaggregator.DoubleStatistics;
+import io.dockstore.openapi.client.model.ExecutionTimeMetric;
+import io.dockstore.openapi.client.model.Metrics;
+import io.dockstore.openapi.client.model.RunExecution;
+import io.dockstore.openapi.client.model.TaskExecutions;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Aggregate Execution Time metrics by calculating the minimum, maximum, and average.
+ */
+public final class ExecutionTimeAggregator implements RunExecutionAggregator<ExecutionTimeMetric, String> {
+    @Override
+    public String getMetricFromRunExecution(RunExecution runExecution) {
+        return runExecution.getExecutionTime();
+    }
+
+    @Override
+    public ExecutionTimeMetric getMetricFromMetrics(Metrics metrics) {
+        return metrics.getExecutionTime();
+    }
+
+    @Override
+    public Optional<RunExecution> getWorkflowExecutionFromTaskExecutions(TaskExecutions taskExecutionsForOneWorkflowRun) {
```

Comment:
This is the new function that aggregates tasks into a workflow execution for the execution time metric